gnosis/safe/safe_creation_tx.py

```diff
@@ -134,14 +132,14 @@ def _calculate_gas(owners: List[str], safe_setup_data: bytes, payment_token: str
 
         base_gas = 60580  # Transaction standard gas
 
-        # TODO If we already have the token, we don't have to pay for storage, so it will be just 5K instead of 20K.
+        # If we already have the token, we don't have to pay for storage, so it will be just 5K instead of 20K.
         # The other 1K is for overhead of making the call
         if payment_token != NULL_ADDRESS:
-            payment_token_gas = 21000
+            payment_token_gas = 55000
```

Comment:
55k? Is that on purpose?

Reply:
Yes, it was working because of the overestimation. First ERC20 transfer is very expensive, and 55K is what we used in CREATE2 creation, so I've just used the same

Reply:
For me this feels extremely high and is completely counter intuitive when reading the comment above. We can assume that we normally own some tokens of the possible payment tokens.

This function contains a lot of magic values that are left unexplained (e.g. `base_gas = 60580` ... not sure what standard gas this is, looking at the comment I would have assumed `21000`, `+ 270000` ... not sure where this is coming from). This makes reviewing this pretty hard :/

Reply:
Ahh this is the deprecated creation flow anyways, right? In this case it is not super important